solution/flask_apps/models/Applications.py

```
@@ -0,0 +1,91 @@
+from ..services.db.db_handler import PostgresDBService
+import psycopg2
+
+import pandas as pd
+
+from .AbstractModel import AbstractModel
+
+
+class Applications:
+
+    def __init__(self, person_id, class_id, date):
+        self.id = None
+        self.person_id = person_id
+        self.class_id = class_id
+        self.date = date
+
+class ApplicationsSql(AbstractModel):
+
+    db_handler = PostgresDBService()
+
+    def save(self, application_object):
+
+        application_exists = self.get_by_id(application_object.id)
+
+        if application_exists:
+            print({"message": "Application aleady exists"})
+        else:
+            save_sql = """
+                INSERT INTO applications(person_id, class_id, date) VALUES\
+                (%s, %s, %s) RETURNING id, person_id, class_id, date
+            """
+            application_params = (application_object.person_id, application_object.class_id,
+                             application_object.date)
+
+            application_obj = self.db_handler.insert_update(save_sql, application_params)
+
+            if not application_obj:
+                print("Could not add applications", psycopg2.DatabaseError)
```

you should throw an exception and stop execution not just printing the error